portfolio/src/main/webapp/script.js

```diff
@@ -187,3 +187,22 @@ function loadPage(pageNumber) {
   }
   page = pageNumber;  
 }
+
+function checkLogin() {
+  fetch('/login').then(response => response.json()).then(status => {
+    const postAndDeleteElement = document.getElementById('post-and-delete');
+    if(status[0] === 'false'){
```

Comment:
This will (hopefully) change to a named field (something like status.loggedIn) of boolean type, which will simplify this if-statement.